lessons/8_loops/index.html

```diff
@@ -0,0 +1,308 @@
+                console.log(items[3]);
+            </code></pre>
+        </section>
+
+        <section>
+          <h5>
+            BUT... what happend if the array have 30 or even more items 🤯
+          </h5>
+          <img src="./noo.gif" width="180" height="130" />
+          <br />
+          <pre class="fragment"><code data-trim class="hljs">
+                const items = ["Item 1", "Item 2", "Item 3", ..., "Item 30"];
+                console.log(items[0]);
+                console.log(items[1]);
+                .
+                .
+                .
+                console.log(items[29]);
+            </code></pre>
+        </section>
+
+        <section>
+          <h3>Don't worry exist a better solution for that!! 🎉</h2>
+          <h1 class="fragment">LOOPS</h3>
+        </section>
+
+        <section>
+            <h3>What is Loops?</h3>
+          <ul>
+            <li>
+             Loops is mainly used for iterating over arrays or objects if you have to run the same code several times.
+            </li>
+             
+          </ul>
+        </section>
+  
+        <section>
+            <pre><code data-trim class="" >
+                const items = ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6", "Item 7"];
+                //to print the items always is running items[0] -> just the index is changing
+                console.log(items[0]);  // index: 0
+                console.log(items[1]);  // index: 1
+                console.log(items[2]);  // index: 2
+                console.log(items[3]);  // index: 3
+                console.log(items[4]);  // index: 4
+                console.log(items[5]);  // index: 5
+                console.log(items[6]);  // index: 6
+            </code></pre>
+            <br/>
+            <h2 class="fragment">Solution</h2>
+            <pre class="fragment"><code data-trim class="hljs">
+                const items = ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6", "Item 7"];
+                for (let i = 0; i < items.length; i++) {
+                    console.log(items[i]);
+                }
+            </code></pre>
+        </section>
+
+        <section>
+            <h3>For Loop structure</h3>
+            <pre><code data-trim class="hljs">
+                for (let i = 0; i < 10; i++) {
+                  // code block to be executed
+                }
+                for ([1. initialization]; [2. condition]; [3. increment]){
+                  // code block to be executed
+                }
+            </code></pre>
+            <div>
+                <ol>
+                    <li class="fragment"><b><i>let i= 0:</i></b> Initializes the counter variable i.</li>
+                    <li class="fragment"><b><i>i < 10:</i></b> Defines the condition for executing the code block.</li>
+                    <li class="fragment"><b><i>i++:</i></b> updates the loop counter (i) with a new value each time the loop runs.</li>
```

Comment:
```suggestion
                    <li class="fragment"><b><i>i++:</i></b> increments the loop counter (i) each time the loop runs.</li>
```
Only from a js perspective, if it would be a new value you would be able to use `const` as an index in a loop. The variable gets reassigned that's why I think new value is misleading.